extensions/lifecycle/conductor/LifecycleConductor.js

```diff
@@ -130,6 +154,100 @@ class LifecycleConductor {
         ], () => {});
     }
 
+    _controlBacklog(done) {
+        if (!this.lcConfig.conductor.backlogControl.enabled ||
+            !this.lcConfig.backlogMetrics) {
+            return process.nextTick(done);
```

Comment:
Since `_controlBacklog` is called within an `async.waterfall`, would returning `done()` here work as well?

Reply:
Would work indeed, though I think it's better to stay safe, i.e. be consistent in the scope of a single function (in case this function would later be called from a sync context).

Reply:
And performance here is a secondary concern.